analyzer/language/ruby/gemspec/gemspec.go

```diff
@@ -0,0 +1,51 @@
+package gemspec
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/analyzer/language"
+	"github.com/aquasecurity/fanal/types"
+	"github.com/aquasecurity/go-dep-parser/pkg/ruby/gemspec"
+	godeptypes "github.com/aquasecurity/go-dep-parser/pkg/types"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&gemspecLibraryAnalyzer{})
+}
+
+const version = 1
+
+var fileregex = regexp.MustCompile(`.*/specifications/.+.gemspec`)
+
+type gemspecLibraryAnalyzer struct{}
+
+func (a gemspecLibraryAnalyzer) Analyze(target analyzer.AnalysisTarget) (*analyzer.AnalysisResult, error) {
+	r := bytes.NewReader(target.Content)
+	parsedLib, err := gemspec.Parse(r)
+	if err != nil {
+		return nil, xerrors.Errorf("failed to parse %s: %w", target.FilePath, err)
+	}
+
+	if parsedLib.Name == "" {
+		return nil, nil
+	}
```

Comment:
This is already validated
https://github.com/aquasecurity/go-dep-parser/blob/main/pkg/ruby/gemspec/parse.go#L88-L90